test/resources/microshift-network.resource

```diff
@@ -29,12 +33,13 @@ Expose Hello MicroShift Pod Via NodePort
 
 Access Hello Microshift
     [Documentation]    Try to retrieve data from the "hello microshift" service end point
-    [Arguments]    ${ushift_port}    ${ushift_ip}=${USHIFT_HOST}
+    [Arguments]    ${ushift_port}    ${ushift_ip}=${USHIFT_HOST}    ${path}=${EMPTY}
 
     ${connect_to}=    Set Variable    "hello-microshift.cluster.local:${HTTP_PORT}:${ushift_ip}:${ushift_port}"
+    ${url_path}=    Set Variable    "http://hello-microshift.cluster.local${path}"
 
     ${result}=    Run Process
-    ...    curl -i http://hello-microshift.cluster.local --connect-to ${connect_to}
+    ...    curl -i ${url_path} --connect-to ${connect_to}
```

Comment:
At some point it would be good to test without the `--connect-to` flag to ensure that we're advertising the service correctly via mDNS. cc @pmtk